nss-day-cohort-29/welcome-to-nashville-cyclones
# Added Zomato and Eventbrite data to local JSON database

# Description
Added Zomato and Eventbrite data to local JSON database, in case their APIs get mad at us again.

## Type of change

Please delete options that are not relevant.


# Testing Instructions for Change Made

1. `git fetch --all`
1. `git checkout meg-zomato-added-to-db`
1. run grunt
1. Go to http://localhost:8088/ and make sure that both databases are there


# Checklist:
